### PR TITLE
Add poster template and update README with information on how to use it.

### PR DESCRIPTION
**Pull request by nsryan2**
## Summary of changes

This PR adds a a template for a poster which is based on Gwen's repository [2019-chee-wm-poster](https://github.com/arfc/2019-chee-wm-poster). It also updates the README of this repository to include information on how to use the template, and a link to the [ARFC Poster Manual](http://arfc.npre.illinois.edu/manual/guides/writing/poster/).

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] I have read the **CONTRIBUTING** document.
- [x] My code follows the code style of this project.
- [x] My change requires a change to the documentation.
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

## Associated Issues and PRs

- Issue: arfc/arfc.github.io#160

## Associated Developers

- Dev: @Dayvis7 

## Checklist for Reviewers

Reviewers should use [this link](/manual/guides/pull_requests) to get to the 
Review Checklist before they begin their review.

### CODE REVIEW

**Comment by nsryan2** [comment]
Once this PR is approved and merged arfc/arfc.github.io#160 should be completed.

File: README.md [hunk 1]
```diff
@@ -1 +1,124 @@
...
+The element in {call} will be what you type to use the acronym, whereas {output} will be what 
+compiles for every use after the first time you call the acronym.
+
+# Using an Acronym:
+This process of adding and using an acronym may seem tedious, but if you call the acronym every 
+time you wish to use it, LaTex will make sure it is defined. Meaning the first time you use it, the 
+acronym will be attached to its meaning, and that every subsequent use is of the acronym itself.
+
+To call an existing acronym, all you have to do is insert:
+`\gls{call}`
```
Comment (gwenchee) [inline README.md:56]: Might be good to add that if you want the plural form (it just adds an s), you can use \gls{pl}

**Review by gwenchee — APPROVED** [review]
I cloned `nsryan2/poster-template` and both the abstract and poster directories built the PDFs successfully. Once you address my very minor comment in the README, we can merge. Good job @nsryan2!

**Comment by gwenchee** [comment]
Awesome! I'll merge

**Pull request merged**